cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -21,15 +21,17 @@ desiredState:
         dnsPolicy: "Default"  # Don't use cluster DNS.
         containers:
           - name: etcd
-            image: quay.io/coreos/etcd:v2.0.3
+            image: gcr.io/google_containers/etcd:2.0.9
```

Comment:
What @thockin said. And yes, I'm using quay.io. Thanks
On May 5, 2015 18:31, "Tim Hockin" <notifications@github.com> wrote:

> In cluster/addons/dns/skydns-rc.yaml.in
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7675#discussion_r29692032>
> :
>
> > @@ -21,15 +21,17 @@ desiredState:
> >          dnsPolicy: "Default"  # Don't use cluster DNS.
> >          containers:
> >            - name: etcd
> > -            image: quay.io/coreos/etcd:v2.0.3
> > +            image: gcr.io/google_containers/etcd:2.0.9
>
> Where is the Dockerfile for this? What is significant about our releease
> vs upstream?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/7675/files#r29692032>
> .
>